kotlin/samples/tictactoe/common/src/main/java/com/squareup/sample/panel/PanelScreen.kt

```diff
@@ -0,0 +1,24 @@
+package com.squareup.sample.panel
+
+import com.squareup.viewregistry.BackStackScreen
+import com.squareup.viewregistry.ModalContainerScreen
+
+/**
+ * Shows an optional stack of modal [PanelScreen]s over a base screen.
+ */
+typealias PanelContainerScreen<B, T> = ModalContainerScreen<B, PanelScreen<T>>
+
+/**
+ * Wraps a screen of a modal subflow in the Tic Tac Workflow sample app.
```

Comment:
This is a wart. It exists solely to give us a hook to register `PanelContainer` against.